dll/win32/kernel32/kernel32.spec

```
@@ -543,7 +543,7 @@
 @ stdcall GetNumberOfConsoleMouseButtons(ptr)
 @ stdcall GetOEMCP()
 @ stdcall GetOverlappedResult(long ptr ptr long)
-@ stub -version=0x600+ GetPhysicallyInstalledSystemMemory
+@ stdcall -version=0x600+ GetPhysicallyInstalledSystemMemory(long ptr)
```

Parameter should be just "ptr" (one single pointer).
```suggestion
@ stdcall -version=0x600+ GetPhysicallyInstalledSystemMemory(ptr)
```